resources/views/page/posts/create.blade.php

```diff
@@ -0,0 +1,38 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="wrap-create-post-page">
+    <h2 class="text-center">{{ trans('post.title_create') }}</h2>
+    <form action="{{ route('posts.store') }}" method="POST" enctype="multipart/form-data">
+        @csrf
+        <div class="form-group">
+            <label>{{ trans('post.title') }}</label>
+            <input type="text" class="form-control" name="title" required>
+            @if ($errors->has('title'))
```

Comment:
```suggestion
            @errors('title')
```
thao khảo cách này nhé